superset/common/query_context_processor.py

```diff
@@ -522,34 +545,92 @@ def processing_time_offsets(  # pylint: disable=too-many-locals,too-many-stateme
                         offset, outer_to_dttm
                     )
 
+                    # For relative offsets, keep the original behavior
```

Comment:
Not quite sure why altering the existing behavior, if we go this path then the FF doesn't guarantee we're not affecting the existing API